src/infrastructure/validation/logger/LoggerFactory.java

```diff
@@ -14,9 +13,6 @@
 	 */
 	private static ILogger loggerManager;
```

Comment:
Hanging */

Reply:
doesnt seem to be the case at my end. Can you recheck?

Reply:
Checked. Its not hanging. I misread it.